bftengine/src/bftengine/RequestHandler.cpp

```diff
@@ -114,6 +114,11 @@ void RequestHandler::execute(IRequestsHandler::ExecutionRequestsQueue& requests,
         LOG_WARN(GL, "Received a Tick, but the cron table registry is not initialized");
         req.outExecutionStatus = static_cast<uint32_t>(OperationResult::INTERNAL_ERROR);
       }
+    } else if (req.flags & MsgFlag::CONSENSUS_PP_FLAG) {
```

Comment:
if I understood correctly, you need to filter this request because you attach a client request to the consensusPP message. But I'm not sure you have to do it (maybe we can simply have a flag inside the PP message itself)